src/routes/features/+page.svelte

```diff
@@ -59,7 +65,7 @@
       <div class="section-title">
         <h2>Pro</h2>
         <p>$3/month</p>
-        <a class="btn" style="color: black; border-color: black;" href="/account?action=upgrade">Upgrade</a>
+        <button class="btn" style="color: black; border-color: black;" on:click={upgrade}><h3>Upgrade</h3></button>
```

Comment:
I see, but I think it would be better to attach `onClick` to an `a` tag to keep it semantically correct.

Reply:
okay ! will make that change then

Reply:
okay so i changed the sign up link back to an anchor tag, but when changing this button to be an anchor with no `href`, a11y yells at me quite a bit lol. i think it needs to be a button so that the click handler automatically picks up any keyboard interactions.